automatminer_dev/tasks/bench.py

```diff
@@ -145,7 +145,7 @@ def run_task(self, fw_spec):
 
         # Evaluate model
         true = result_df[target]
-        test = result_df[target + " predicted"]
+        test = result_df[learner.target_output_col]
```

Comment:
Since you are making changes to things which affect `_dev`, this will take me some time to merge, since the entire infrastructure I use to run AMM benchmarks depends on `dev`.

Reply:
No problem.